code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -106,6 +106,13 @@
 		"yellow",
 	)
 
+	var/static/list/tier_one_colours = list(
```

Comment:
While the name is self-explanatory, give this a documenting comment anyway for our metrics (explanation for variable preceded by `///`)

Reply:
got it!

Reply:
Added the following comment: ``///A random slime that is blue, metal, orange, or purple.``

Reply:
Jacquerel was asking that you document this variable, which you still need to do.

Also you should move this variable to the tier_one subtype instead of on all slimes.

Reply:
Just did that in latest commit.